mars_form_app/src/components/Form.js

```diff
@@ -0,0 +1,321 @@
+import React, {Component} from 'react';
+// import axios from 'axios';
+import countries from "./countries"
+
+class Form extends Component{
+    state = {
+        name: "",
+        birthday: "",
+        countries: countries,
+        selectedCountry: "United States",
+        diet: "",
+        explorer: "",
+        formCompleted: false,
+        formSubmitted: false,
+        breathe:"no",
+        married:false,
+        situation:"",
+        claustrophobic:false,
+        familyHistory:{},
+        living:{},
+        education:{}
+    }
+
+    // populateCountry() {
+    //     // e.preventDefault();
+    //     axios.get("http://vocab.nic.in/rest.php/country/json").then(res => {
+    //         let {countries} = res.data;
+    //         // debugger
+    //         // this.setState({countries});
+    //     })
+    // }
+    handleChange = (e) => {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+    handleSubmit = (e) => {
+        e.preventDefault();
+        this.setState({
+            formCompleted: !this.state.formCompleted
+        })
+        if(this.state.formCompleted === true) {
+            let div = document.querySelector("#formSubmitted");
+            div.hidden = false;
+        }
+    }
```

Comment:
I have a feeling this may not be working the way you want it too. When you call setState you are  doing an asynchronous operation. Therefore you won't be able to count on state being what you want it to be in the next line which is synchronous.  Also don't use querySelector. We NO longer need to do that. QuerySelector helps us manipulate the DOM directly. A main use of React is that we will have React update the DOM for us. We update Reacts virtual DOM and it handles the update to the DOM.